src/core/ConfigManager.cpp

```diff
@@ -369,15 +369,14 @@ void ConfigManager::loadConfigFile()
 		m_flDir = ensureTrailingSlash( QDir::home().absolutePath() );
 	}
 
-	if( m_ladDir.isEmpty() || m_ladDir == QDir::separator() ||
-			( !m_ladDir.contains( ':' ) && !QDir( m_ladDir ).exists() ) )
+	if( m_ladDir.isEmpty()  )
 	{
 #if defined(LMMS_BUILD_WIN32)
 		m_ladDir = qApp->applicationDirPath() + "/plugins/ladspa" + QDir::separator();
 #elif defined(LMMS_BUILD_APPLE)
 		m_ladDir = qApp->applicationDirPath() + "/../lib/lmms/ladspa/";
 #else
-		m_ladDir = qApp->applicationDirPath() + '/' + LIB_DIR + "/ladspa/";
+		m_ladDir = qApp->applicationDirPath() + '/' + LIB_DIR + "lmms/ladspa/";
```

Comment:
> Is this where we set it?

I belive so, 

> but should it?

IMO paths end with a trailing / (separator),  filenames don't.

We could change ````LIB_DIR```` to not include the trailing separator, but that would involve changes across the codebase, outside the scope of this pull request.  I am happy to work on that as a separate PR if required?